File: library/src/main/java/su/levenetc/android/textsurface/animations/Parallel.java
package su.levenetc.android.textsurface.animations;

import su.levenetc.android.textsurface.contants.TYPE;
import su.levenetc.android.textsurface.interfaces.ISurfaceAnimation;

/**
 * Created by Eugene Levenetc.
 */
public class Parallel extends AnimationsSet {
	public Parallel(ISurfaceAnimation... animations) {
		super(TYPE.PARALLEL, animations);
	}
}

File: library/src/main/java/su/levenetc/android/textsurface/animations/Parallel.kt
package su.levenetc.android.textsurface.animations

import su.levenetc.android.textsurface.contants.TYPE
import su.levenetc.android.textsurface.interfaces.ISurfaceAnimation

/**
 * Created by Eugene Levenetc.
 */
class Parallel(vararg animations: ISurfaceAnimation?) : AnimationsSet(TYPE.PARALLEL, *animations)